varnam-wrapper.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+export VARNAM_SUGGESTIONS_DIR=~/.local/share/varnam/suggestions/
```

Comment:
Why you need this?

Reply:
It's a wrapper script to make the suggestions folder in host because of this : https://github.com/flathub/flathub/pull/1747#discussion_r470964069

I guess I should move these and the appdata files to upstream